Resources/Prototypes/Research/industrial.yml

```diff
@@ -183,6 +183,7 @@
     - OreBagOfHolding
     - MiningDrillDiamond
     - AdvancedMineralScannerEmpty
+    - ClothingShoesBootsJump
```

Comment:
The tech does not really have anything to do with excavation, except that it is supposed to be used by salvagers. MAybe put it into another technology?

Reply:
> The tech does not really have anything to do with excavation, except that it is supposed to be used by salvagers. Maybe put it into another technology?

Yea, I thought about which one, but nothing better than this I did not find, to create a new technology for the sake of boots was not very rational, and to throw in T1 technology Salvage Equipment considered somehow not balanced, and in it and so it is too much things are contained
So if you have any suggestions on where to put them, I'll listen to them, because I don't have any ideas

Reply:
Although I'll probably put them in the “Salvage Equipment” tech but make the recipe a little more expensive